### DISCUSSION TITLE
Swagger UI not working in Fastapi version 0.61.2

### DISCUSSION THREAD

**Discussion by meshachdamilare** [fastapi/fastapi#7027] [Questions]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [x] I already read and followed all the tutorial in the docs and didn't find an answer.
* [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

### Example

Here's a self-contained, [minimal, reproducible, example](https://stackoverflow.com/help/minimal-reproducible-example) with my use case:

```Python
from fastapi import FastAPI

app = FastAPI()


@app.get("/")
def read_root():
    return {"Hello": "World"}
```

### Description

* Open the browser and call the endpoint `/`.
* It returns a JSON with `{"Hello": "World"}`.
* But I expected it to return `{"Hello": "Sara"}`.

### Environment

* OS: [e.g. Linux / Windows / macOS]:
* FastAPI Version [e.g. 0.3.0]:

To know the FastAPI version use:

```bash
python -c "import fastapi; print(fastapi.__version__)"
```

* Python version:

To know the Python version use:

```bash
python --version
```

### Additional context

Just installed fastAPI 0.61.2 version yet the swagger UI isn't working

**Comment by meshachdamilare** [comment]
Just installed fastAPI 0.61.2 version yet the swagger UI isn't working

**Comment by Mause** [comment]
Please provide more details, we cannot help you with the details you have provided

**Comment by meshachdamilare** [comment]
Anytime I ran my main:app and navigate to 127.0.0.1:8000/docs, I got blank page on my browser.

**Comment by Mause** [comment]
Is there an error in your server console, or in the Javascript console in your browser?

**Comment by ycd** [comment]
Tried to reproduce this but everything works 

Here are my versions.

```bash
$ pip list | grep -E "fastapi|starlette|uvicorn"

fastapi           0.61.2    
starlette         0.14.1    
uvicorn           0.11.8    
```

Please send a curl request to the /docs and share the headers

```bash
curl -v http://localhost:8000/docs
```

**Comment by meshachdamilare** [comment]
> Tried to reproduce this but everything works
> 
> Here are my versions.
> 
> ```shell
> $ pip list | grep -E "fastapi|starlette|uvicorn"
> 
> fastapi           0.61.2    
> starlette         0.14.1    
> uvicorn           0.11.8    
> ```
> 
> Please send a curl request to the /docs and share the headers
> 
> ```shell
> curl -v http://localhost:8000/docs
> ```

Thanks. It worked.

**Comment by Mause** [comment]
If your issue is solved, can you please close it?

**Comment by meshachdamilare** [comment]
> If your issue is solved, can you please close it?

Yes, it has been solved.

**Comment by raphaelauv** [comment]
@Christomesh so could you please close this issue.

**Comment by tiangolo** [comment]
Thanks for the help here everyone! :clap: :bow: 

Thanks for reporting back and closing the issue :+1:
